src/ui/analyse/AnalyseCtrl.ts

```diff
@@ -90,7 +90,8 @@ export default class AnalyseCtrl {
     orientation: Color,
     shouldGoBack: boolean,
     ply?: number,
-    tab?: number
+    tab?: number,
+    tabId?: string
```

Comment:
Done. I could not find an instance of the `tab` parameter actually being used when routing to the analysis board. I swapped it out for the `tabId` parameter as you suggested so that attribute will be there in the future if we need it.